packages/libs/error-stack/tests/compiletest.rs

```diff
@@ -0,0 +1,7 @@
+#[cfg_attr(not(nightly), ignore = "Outputs are different on toolchains")]
```

Comment:
I don't think this ignore message quite makes sense. Do you mean different _across_ toolchains or something? (as in between different toolchains)

Reply:
Yes, it's *across* toolchains: 8a97f9b7006fbbb83cf834306785a69312f3239d